### PR TITLE
Add missing options to selectize config

### PR DESCRIPTION
**Pull request by topaxi**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/selectize/selectize.js/blob/master/docs/usage.md
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Comment by RyanCavanaugh** [comment]
This PR has been open and unchanged 5 days without signoff or complaint. This will be merged by a maintainer soon if there are no objections.

File: types/selectize/index.d.ts
Comment (DanielRosenwasser) [inline types/selectize/index.d.ts]: Shouldn't this just be `string[]`?

**Review by DanielRosenwasser — CHANGES_REQUESTED** [review]

**Comment by RyanCavanaugh** [comment]
@topaxi Please address comments from the code reviewers.

**Comment by topaxi** [comment]
Search field can be a single string according to
https://github.com/brianreavis/sifter.js/blob/master/lib/sifter.js#L314

Can update the PR to reflect the official selectize documentation.

**Review by DanielRosenwasser — APPROVED** [review]

**Comment by DanielRosenwasser** [comment]
Sorry for the delay, thanks for your patience!

**Pull request merged**
